build.gradle

```diff
@@ -81,24 +81,15 @@ test {
 
 task functional(type: Test) {
   environment 'AZURE_SERVICE_BUS_CONNECTION_STRING','Endpoint=sb://ccd-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=SendAndReceiveCCDMessage;SharedAccessKey=BXOmN0uU5RZN7KW51soNUv7xNBcNtXqwh+ASbD8y4p4=;EntityPath=st-ccd-case-event'
```

Comment:
I think this needs to be updated to point to our equivalent access key. Though we don't have one in demo yet.
https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/sptribs-aso-preview-rg/providers/Microsoft.ServiceBus/namespaces/sptribs-sb-preview/saskey

Reply:
Decided to investigate further in [DTSSTCI-577](https://tools.hmcts.net/jira/browse/DTSSTCI-577).